### ISSUE TITLE
output variables for Coil:WaterHeating:AirToWaterHeatPump:Wrapped in documentation incorrect

### ISSUE THREAD

**Issue by JanghyunJK** [NREL/EnergyPlus#9769]
Issue overview
--------------
Maybe not the right place to report, but

I think, documentation of [output variables](https://bigladdersoftware.com/epx/docs/22-1/input-output-reference/group-heating-and-cooling-coils.html#outputs-24) for [Coil:WaterHeating:AirToWaterHeatPump:Wrapped](https://bigladdersoftware.com/epx/docs/22-1/input-output-reference/group-heating-and-cooling-coils.html#coilwaterheatingairtowaterheatpumpwrapped) is incorrect. Changing the term "Heating" in output variables to "Cooling" seems to work.

### Details
- so I tried with what we have in the documentation and was getting this message:
```
************* Simulation Error Summary *************
** Warning ** The following Report Variables were requested but not generated -- check.rdd file**   ~~~   ** Either the IDF did not contain these elements, the variable name is misspelled,
**   ~~~   ** or the requested variable is an advanced output which requires Output : Diagnostics, DisplayAdvancedReportVariables;
************* Key=*, VarName=HEATING COIL TOTAL HEATING RATE, Frequency=TimeStep
************* Key=*, VarName=HEATING COIL TOTAL WATER HEATING RATE, Frequency=TimeStep
************* Key=*, VarName=HEATING COIL WATER HEATING ELECTRICITY RATE, Frequency=TimeStep
```
- and checked rdd file and found these instead:
![image](https://user-images.githubusercontent.com/26775789/207891025-de75a2c4-1d37-4476-a6c2-3b37258d1ed0.png)

- and tried with "Cooling" term in the output variable names (instead of "Heating") and it was grabbing (I believe) the right output variables such as:
  - 50gal_heat_pump_water_heater_-_5kbtu/hr_coil_cooling_coil_total_water_heating_rate   [W] 
  - 50gal_heat_pump_water_heater_-_5kbtu/hr_coil_cooling_coil_water_heating_electricity_rate   [W]

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by shorowit** [comment]
This was already fixed, see the 22.2 documentation:
https://bigladdersoftware.com/epx/docs/22-2/input-output-reference/group-heating-and-cooling-coils.html#outputs-24

**Comment by JanghyunJK** [comment]
oh by d0.1 version.
